src/Playwright.Tests/PageNetworkResponseTests.cs

```diff
@@ -174,5 +174,27 @@ public async Task ShouldWaitUntilResponseCompletes()
             serverResponseCompletion.SetResult(true);
             Assert.AreEqual("hello world!", await responseText);
         }
+
+        [PlaywrightTest("har.spec.ts", "should return security details directly from response")]
+        [Test, SkipBrowserAndPlatform(skipLinux: true, skipFirefox: true)]
```

Comment:
upstream its webkit+linux (and below). Whats the reason that you are skipping it for FF?

Reply:
Typo. Good spot.